demos/_demo-data/create-db.php

```diff
@@ -19,12 +19,13 @@
 /** @var Persistence\Sql $db */
 require_once __DIR__ . '/../init-db.php';
 
-if (getenv('GITHUB_JOB') === 'unit-test') {
+if (getenv('GITHUB_JOB') !== false && strpos(getenv('GITHUB_JOB'), 'unit-') === 0) {
     echo "skip db creation in create-db\n\n";
 
     return;
 }
 
-(new Migrator(new Dummy($db)))->create();
+$migr = new Migrator(new Dummy($db));
+$migr->dropIfExists()->create();
```

Comment:
this is exactly what we do not want, we want to be sure this skip is called only once on a clean testing DB